scripts/ci/_utils.sh

```diff
@@ -26,6 +26,8 @@ declare -a EXTRA_DOCKER_PROD_BUILD_FLAGS
 export EXTRA_DOCKER_FLAGS
 export EXTRA_DOCKER_PROD_BUILD_FLAGS
 
+declare -a files_to_cleanup_on_exit=()
```

Comment:
This should be UPPERCASE. The convention which I stick to everywhere else (And it's good to keep it consistent) is to keep variable names as UPPERCASE and function names lowercase. This is traditional approach and makes things much more obvious.

Reply:
I'll keep with the convention then.


The reason I've made it an array is for the next tidyup commit around the cache directory (We're blanket deleting the cache folder, ignoring the 'SKIP_CACHE_DELETION' flag and calling `remove_cache_directory` unconditionally in script_end).

Reply:
FWIW https://google.github.io/styleguide/shellguide.html#function-names disagrees with this style guide:

> Function Names
>
> Lower-case, with underscores to separate words. 
>
> ...
>
> Variable Names
>
> As for function names.
> 
>
> Constants and Environment Variable Names
> 
> All caps, separated with underscores, declared at the top of the file.
>
> Constants and anything exported to the environment should be capitalized.

This variable is neither a constant, nor is it exported.

Reply:
https://github.com/apache/airflow/pull/9105/commits/9d14f96ff12fd5834c859d7126bc11af92ffb2ac to update to current style

Reply:
I am ok with adopting a style guide for that if we are to adopt it consistently (including some dev scripts).  Maybe that's the right time to agree on it. 

I think so far there was no convention whatsoever in many places. I am not tied to any convention and happy to adapt as long as we consciously agree do it and stick to it everywhere  :).

I love to change my habits from time to time. Shall we ask people? I think also that might be a good time to discuss pros/cons of using bash or maybe adopt another approach in the future.  I can think of two approaches - python as you suggested or go as it produces statically linked multiple-platform binaries that you can downioad/install separately without dependencies and is already used in docker and many other tools like that). 

So far we discussed it in slack. But maybe let's finish some of the important things first like images and chart - not too open too many discussions and work in parallel. We all already have too much on our plates.

Reply:
Yeah, if we're changing 
I think for breeze/CI shell scripts it's mostly you and I who write the shell scripts, and so long as we document 

> > Curious, I wonder why that precommit check didn't run for me locally.
> > Oh cos of the files regex -- which skips it unless one of the breeze files is changed.
> 
> Yep. We run pre-commit with `--all-files` in PR just in case there are some hidden dependencies.

Smart man. Cos there totally were here :D

Reply:
Yeah, I'm a fan of making change of style the only thing in a PR.

Honestly, I think you and I are largely the only ones who work in any depth in the scripts/breeze (and mostly you right now, but I'm trying to change that a bit), so I'd probably say we decide on and document a style, then tag a the usual suspects on a PR would be enough -- (unlike the python side which affects many more people).